### PR TITLE
Removed Arwen OOP option

### PR DESCRIPTION
**Pull request by iulianpascalau**
- removed Arwen OOP option

### CODE REVIEW

File: cmd/node/config/config.toml [hunk 1]
```diff
@@ -640,33 +640,31 @@
...
+            { StartEpoch = 0, Version = "v1.2" },
+            { StartEpoch = 3, Version = "v1.3" },
+            { StartEpoch = 4, Version = "v1.2" },
+            { StartEpoch = 5, Version = "v1.3" },
+            { StartEpoch = 6, Version = "v1.2" },
+            { StartEpoch = 7, Version = "v1.3" },
+            { StartEpoch = 8, Version = "v1.2" },
+            { StartEpoch = 9, Version = "v1.3" },
         ]
         [VirtualMachine.Querying.OutOfProcessConfig]
```
Comment (sasurobert) [inline cmd/node/config/config.toml:669]: Delete out of process config as well

File: process/factory/shard/vmContainerFactory.go [hunk 2]
```diff
@@ -258,16 +242,6 @@ func (vmf *vmContainerFactory) createInProcessArwenVMByVersion(version config.Ar
 	}
 }
 
-func (vmf *vmContainerFactory) createOutOfProcessArwenVMByVersion(version config.ArwenVersionByEpoch) (vmcommon.VMExecutionHandler, error) {
-	logVMContainerFactory.Debug("createOutOfProcessArwenVM", "version", version)
-	switch version.Version {
-	case "v1.2":
-		return vmf.createOutOfProcessArwenVMV12()
```
Comment (sasurobert) [inline process/factory/shard/vmContainerFactory.go:265]: Delete function create out of process arwen v12

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: process/factory/shard/vmContainerFactory.go (hunk 2 above)
Comment (iulianpascalau) [reply process/factory/shard/vmContainerFactory.go:265]: done

File: cmd/node/config/config.toml (hunk 1 above)
Comment (iulianpascalau) [reply cmd/node/config/config.toml:669]: done

**Review by camilbancioiu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
image

**Pull request merged**